runestone/parsons/js/parsons.js

```diff
@@ -2042,8 +2042,10 @@ export default class Parsons extends RunestoneBase {
         //this.helpButton.disabled = true;
         //}
         // if less than 3 attempts
+        console.log("msg_parson_atleast_three_attempts")
```

Comment:
If these are for debugging then please remove them.

Reply:
Removed those lines